packages/core/src/modules/attendance/attendance-service.ts

```diff
@@ -109,10 +116,24 @@ export class AttendanceServiceImpl implements AttendanceService {
   async updatePool(poolId: AttendancePoolId, data: Partial<AttendancePoolWrite>) {
     const pool = await this.attendanceRepository.getPoolById(poolId)
 
+    // FIXME: refactor into own method for changing capacity to make this easier to manage.
     if (data.capacity && pool.numAttendees > data.capacity) {
       throw new AttendanceValidationError("Cannot change pool capacity to less than the reserved spots")
     }
 
+    if (data?.capacity && data.capacity > pool.capacity) {
+      const newCapacity = data.capacity - pool.capacity
+      const attendees = await this.attendeeService.getByAttendancePoolId(poolId) // These are in order of reserveTime
+
+      const attendeesThatAreNowReserved = attendees.slice(pool.capacity, pool.capacity + newCapacity)
```

Comment:
```suggestion
    if (data?.capacity && data.capacity > pool.capacity) {
      const attendees = await this.attendeeService.getByAttendancePoolId(poolId) // These are in order of reserveTime

      const attendeesThatAreNowReserved = attendees.slice(pool.capacity, data.capacity)
```

Reply:
Could maybe also be nice to rename `data` to `newPool` or something more explicit, but yeah. Not that important